docs/source/markdown/podman.1.md

```diff
@@ -334,18 +426,18 @@ See the `subuid(5)` and `subgid(5)` man pages for more information.
 
 Images are pulled under `XDG_DATA_HOME` when specified, otherwise in the home directory of the user under `.local/share/containers/storage`.
 
-Currently the slirp4netns package is required to be installed to create a network device, otherwise rootless containers need to run in the network namespace of the host.
+Currently the `slirp4netns` package is required to be installed to create a network device, otherwise rootless containers need to run in the network namespace of the host.
```

Comment:
@mheon does this still hold for 3.2?

Reply:
Yes, this will likely always be true, we've just added the additional requirement of CNI on the host for these installations.

Reply:
Thanks for the update, are there any changes that should be made to this section that you can think of?

Reply:
I am leaving this as is